docs/Dashboard.md

```diff
@@ -0,0 +1,44 @@
+
+# Pulsar Dashboard
+
+Pulsar dashboard is a web application that let users monitor
+the current stats for all the topics, representing all
+the information in tabular form.
+
+The dashboard is composed of a "collector" that polls the
+stats from all the brokers in a Pulsar instance (across
+multiple clusters) and store all the information into
+a Postgres database.
+
+Finally a Django web-app is used to render the collected
+data.
+
+## Install
+
+The easiest way to use the dashboard is to run it inside
+a Docker container. A `Dockerfile` to generate the image
+is provided.
+
+To generate the Docker image:
+
+```shell
+$ docker build -t pulsar-dashboard dashboard
```

Comment:
Ok, let us get the link to docker documentation(https://docs.docker.com/engine/installation/). Assuming everyone knows about docker is debatable :)